CMakeLists.txt

```diff
@@ -52,7 +52,16 @@ message ( STATUS "ROOT_VERSION: ${ROOT_VERSION}" )
 
 find_package( Geant4 REQUIRED ) 
 find_package( LCIO REQUIRED)
-
+# Shim for older LCIO versions
+if(NOT TARGET LCIO::lcio)
+    add_library(LCIO::lcio INTERFACE IMPORTED GLOBAL)
+    set_target_properties(LCIO::lcio
+      PROPERTIES
+      INTERFACE_INCLUDE_DIRECTORIES "${LCIO_INCLUDE_DIRS}"
+      INTERFACE_LINK_LIBRARIES "${LCIO_LIBRARIES}"
+    )
+  endif()
```

Comment:
```suggestion
endif()
```

Reply:
Thanks, fixed the indentation!

It builds with the latest release, which was my main conern.
(I copied that shim from DD4hep, where it worked for a long time.)